### PR TITLE
remove limits from certificate auth info and export subcommands

### PR DESCRIPTION
**Pull request by bryanchriswhite**
### [V3-1460](https://storjlabs.atlassian.net/browse/V3-1460)
What: Listing certificate signing authorizations and related data was limited to 1000 authorization records. This PR removes that limit.

Why: In order to audit certificate signing authorizations, devops needs to be able to inspect all authorizations.

Thanks for submitting a PR!

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Could the PR be broken into smaller PRs?
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?

### CODE REVIEW

File: pkg/certificates/certificates.go [hunk 1]
```diff
@@ -298,29 +298,54 @@ func (authDB *AuthorizationDB) Get(userID string) (Authorizations, error) {
...
-	keys, err := authDB.DB.List([]byte{}, 0)
-	if err != nil {
-		return nil, ErrAuthorizationDB.Wrap(err)
-	}
-	return keys.Strings(), nil
+func (authDB *AuthorizationDB) UserIDs() (userIDs []string, err error) {
+	err = authDB.DB.Iterate(storage.IterateOptions{
+		First:   []byte{},
+		Recurse: true,
+		Prefix:  storage.Key(""),
```
Comment (egonelbre) [inline pkg/certificates/certificates.go]: You can skip defining `First` and `Prefix` if you don't need them.

File: pkg/certificates/certificates.go [hunk 2]
```diff
@@ -298,29 +298,54 @@ func (authDB *AuthorizationDB) Get(userID string) (Authorizations, error) {
...
-
-	for _, uid := range uids {
-		idAuths, err := authDB.Get(uid)
-		if err != nil {
-			return nil, err
+func (authDB *AuthorizationDB) List() (auths Authorizations, err error) {
+	err = authDB.DB.Iterate(storage.IterateOptions{
+		First:   []byte{},
+		Recurse: true,
+		Prefix:  storage.Key(""),
```
Comment (egonelbre) [inline pkg/certificates/certificates.go]: You can skip defining `First` and `Prefix` if you don't need them.

File: pkg/certificates/certificates.go [hunk 3]
```diff
@@ -298,29 +298,54 @@ func (authDB *AuthorizationDB) Get(userID string) (Authorizations, error) {
...
-	return keys.Strings(), nil
+func (authDB *AuthorizationDB) UserIDs() (userIDs []string, err error) {
+	err = authDB.DB.Iterate(storage.IterateOptions{
+		First:   []byte{},
+		Recurse: true,
+		Prefix:  storage.Key(""),
+	}, func(iterator storage.Iterator) error {
+		listItem := new(storage.ListItem)
+		for {
+			more := iterator.Next(listItem)
```
[the remark this answers is not in the document]
Comment (bryanchriswhite) [reply pkg/certificates/certificates.go]: nice - thanks!

File: pkg/certificates/certificates.go (hunk 1 above)
Comment (bryanchriswhite) [reply pkg/certificates/certificates.go]: good to know - thanks!

**Pull request merged**